devtools/chain/config.toml

```diff
@@ -11,12 +11,12 @@ http_listening_address = "127.0.0.1:8000"
 ws_listening_address = "127.0.0.1:8010"
 maxconn = 25000
 max_payload_size = 1048576
+client_version = "Geth/v1.10.14-unstable-72c2c0ae/linux-amd64/go1.17.5"
```

Comment:
I don't think the version number has to be this value, I think a better way would be to get it like this https://github.com/nervosnetwork/ckb/blob/develop/build.rs#L31

Reply:
Why use geth version here?

Reply:
> Why use geth version here?

Original Ethereum dev chain Web3_ Clientversion returns this value: `geth / v1 10.14-unstable-72c2c0ae/linux-amd64/go1. 17.5`, can I set empty value  in the configuration file or other value？